src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -536,6 +533,11 @@ class BehandlingService(
             ),
         )
 
+        // Lag oppgave for behandling
+        if (!erAutomatisk) {
```

Comment:
feature-toggle-messig - burde vi her hatt en funksjon som sier "erAutomatiskOGFeatureToggleSkruddPå" ? Som "følger oss" gjennom flyten i tilbake?

Reply:
Yes, kan lage det. Tar det i egen PR for enkelhet skyld, da jeg har en lokal branch som bygger på denne.